src/platform/Linux/BLEManagerImpl.h

```diff
@@ -91,51 +139,40 @@ class BLEManagerImpl final : public BLEManager, private BleLayer, private BlePla
     static BLEManagerImpl sInstance;
 
     // ===== Private members reserved for use by this class only.
-
     enum
     {
-        kFlag_AsyncInitCompleted     = 0x0001, /**< One-time asynchronous initialization actions have been performed. */
-        kFlag_AdvertisingEnabled     = 0x0002, /**< The application has enabled CHIPoBLE advertising. */
-        kFlag_FastAdvertisingEnabled = 0x0004, /**< The application has enabled fast advertising. */
-        kFlag_Advertising            = 0x0008, /**< The system is currently CHIPoBLE advertising. */
-        kFlag_AdvertisingRefreshNeeded =
-            0x0010, /**< The advertising state/configuration has changed, but the SoftDevice has yet to be updated. */
+        kFlag_AsyncInitCompleted       = 0x0001, /**< One-time asynchronous initialization actions have been performed. */
+        kFlag_BluezBLELayerInitialized = 0x0002, /**< The Bluez layer has been initialized. */
+        kFlag_AppRegistered            = 0x0004, /**< The CHIPoBLE application has been registered with the Bluez layer. */
+        kFlag_AdvertisingConfigured    = 0x0008, /**< CHIPoBLE advertising has been configured in the Bluez layer. */
+        kFlag_Advertising              = 0x0010, /**< The system is currently CHIPoBLE advertising. */
+        kFlag_ControlOpInProgress      = 0x0020, /**< An async control operation has been issued to the ESP BLE layer. */
+        kFlag_AdvertisingEnabled       = 0x0040, /**< The application has enabled CHIPoBLE advertising. */
+        kFlag_FastAdvertisingEnabled   = 0x0080, /**< The application has enabled fast advertising. */
+        kFlag_UseCustomDeviceName      = 0x0100, /**< The application has configured a custom BLE device name. */
+        kFlag_AdvertisingRefreshNeeded = 0x0200, /**< The advertising configuration/state in ESP BLE layer needs to be updated. */
     };
 
     enum
     {
-        kMaxConnections             = MIN(BLE_LAYER_NUM_BLE_ENDPOINTS, NRF_SDH_BLE_PERIPHERAL_LINK_COUNT),
+        kMaxConnections             = 1,  // TODO: right max connection
```

Comment:
should we have a github issue for these TODOs? Thinking of a TODO format like

TODO(#issueid): ....

so that they are tracked and also greppable when we fix them

Reply:
Yep, I would address it when adding bluez central support where there are multiple ble connection. https://github.com/project-chip/connectedhomeip/issues/2027